crox/src/main.rs

```diff
@@ -163,3 +159,31 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+fn timestamp_to_min_max(timestamp: Timestamp) -> (SystemTime, SystemTime) {
+    match timestamp {
+        Timestamp::Instant(t) => (t, t),
```

Comment:
hmm maybe Instants shall not be handled and filtered out before calling this as it is not part of the output

Reply:
Maybe. I'll leave this as is right now, as it matches the current behavior.